src/sidebar/components/test/search-status-bar-test.js

```diff
@@ -26,174 +27,178 @@ describe('searchStatusBar', () => {
       clearDirectLinkedIds: sinon.stub(),
       clearSelection: sinon.stub(),
     };
-    angular.mock.module('app', {
-      store: fakeStore,
-      rootThread: fakeRootThread,
-    });
-  });
-
-  describe('filterQuery', () => {
-    ['tag:foo', null].forEach(filterQuery => {
-      it('returns the `filterQuery`', () => {
-        fakeStore.getState.returns({ filterQuery });
-
-        const elem = util.createDirective(document, 'searchStatusBar', {});
-        const ctrl = elem.ctrl;
 
-        assert.equal(ctrl.filterQuery(), filterQuery);
-      });
+    SearchStatusBar.$imports.$mock({
+      '../store/use-store': callback => callback(fakeStore),
     });
   });
 
-  describe('filterActive', () => {
-    it('returns true if there is a `filterQuery`', () => {
-      fakeStore.getState.returns({ filterQuery: 'tag:foo' });
-
-      const elem = util.createDirective(document, 'searchStatusBar', {});
-      const ctrl = elem.ctrl;
-
-      assert.isTrue(ctrl.filterActive());
-    });
-
-    it('returns false if `filterQuery` is null', () => {
-      fakeStore.getState.returns({ filterQuery: null });
-
-      const elem = util.createDirective(document, 'searchStatusBar', {});
-      const ctrl = elem.ctrl;
-
-      assert.isFalse(ctrl.filterActive());
-    });
+  afterEach(() => {
+    SearchStatusBar.$imports.$restore();
   });
 
-  describe('filterMatchCount', () => {
-    it('returns the total number of visible annotations or replies', () => {
+  [
+    {
+      description:
+        'reports correct text if 2 annotations match the search fitler',
```

Comment:
```suggestion
        'reports correct text if 2 annotations match the search filter',
```

And the same with a few tests below.